docs/self-hosted-appcircle/configure-server/integrations-and-access/integration.md

```diff
@@ -27,6 +27,7 @@ smtpServer:
   ssl: 'false'
   auth: 'true'
   starttls: 'true'
+  verifyCertificate: 'true'
```

Comment:
This feature is supported for `3.23.1` or later. Version info should be included as an info box